### PR TITLE
Simplify the SS sketches shown as a SB layer -- one layer, not split …

### PR DESCRIPTION
**Pull request by dabreegster**
…by ATF/LCWIP.

#536, and following up to #541

Demo: https://acteng.github.io/atip/sb_ss_one_layer/browse.html

The filters for local schemes are simple:
image

### CODE REVIEW

File: src/lib/browse/schemes/SchemesLayer.svelte
Comment (dabreegster) [inline src/lib/browse/schemes/SchemesLayer.svelte:1]: Mostly just some careful renaming here

File: src/lib/browse/schemes/SchemesLayer.svelte [hunk 1]
```diff
@@ -35,45 +32,37 @@
...
-
-  let localLcwipName = "local_lcwip_schemes";
-  let localLcwipTitle = "Your LCWIP schemes";
-  let localLcwipShow = showHideLayer(localLcwipName);
-  let localLcwipStyle = "interventionType";
-  $: [localLcwipColor, localLcwipLegend] = pickStyle(localLcwipStyle);
+  let localName = "local_schemes";
+  let localTitle = "Your schemes";
+  let localShow = showHideLayer(localName);
+  $: [localColor, localLegend] = pickStyle("interventionType");
```
Comment (Pete-Y-CS) [inline src/lib/browse/schemes/SchemesLayer.svelte:50]: Wondering about local legend: v2 will want to be rebased onto this and we may want to rethink the legend

**Review by Pete-Y-CS — COMMENTED** [review]
Looks good, just one comment that we might wanna rework this after changing to v2?

File: src/lib/browse/schemes/SchemesLayer.svelte (hunk 1 above)
Comment (dabreegster) [reply src/lib/browse/schemes/SchemesLayer.svelte:50]: Yep, let's figure that out in #544. There's more changes to browse needed for v2, like updating the intervention popup

**Pull request merged**